hexa/plugins/connector_accessmod/schema.py

```diff
@@ -289,7 +291,16 @@ def resolve_create_accessmod_file(_, info, **kwargs):
             mime_type=create_input["mimeType"],
             fileset=fileset,
         )
+        fileset.status = FilesetStatus.VALIDATING
```

Comment:
I would move that to the validation worker - until it's picked up by the queue, it should stay in `PENDING` imho